api/models/schema.py

```diff
@@ -281,8 +283,9 @@ def validate_box(cls, v):
             except ValueError:
                 raise ValueError('Bounding box coordinate is not a number')
             item = abs(item)
-            if index % 2 == 0 and item > 180:
-                raise ValueError('Bounding box coordinate must be between -180 and 180')
+            if index % 2 == 0:
```

Comment:
I think these two error messages should indicate witch cardinal direction the coordinate is referring to for each case.

Reply:
Fixed it.